test/test_util_file.cpp

```diff
@@ -304,4 +304,13 @@ TEST(Utils_File_Lock)
     CHECK_NOT(f2.try_rw_lock_exclusive());
 }
 
+TEST(Utils_File_SystemErrorMessage)
+{
+    std::error_code err = std::make_error_code(std::errc::too_many_files_open);
+    std::string_view message = "my message";
+    std::string expected = util::format("%1 (SystemError %2: %3)", message, err.value(), "Too many open files");
```

Comment:
`CHECK_STRING_CONTAINS(e.what(), expected) failed with (my message (SystemError 24: too many files open), my message (SystemError 24: Too many open files))` ...  Only on Windows though.

Reply:
Thanks. Windows seems to provide a slightly different error message, so I've adjusted the tests accordingly.

Reply:
Of course Windows just has to be different